app/src/main/res/values/strings.xml

```diff
@@ -306,60 +306,60 @@
         9. Other memory and crash fixes.
     </string>
 
-    <string name="connection_http3">HTTP3</string>
+    <string name="connection_http3">HTTP/3</string>
 
     <!-- Settings Fragment strings -->
     <string name="settings_ip_network_heading">Choose IP version</string>
```

Comment:
```suggestion
    <string name="settings_ip_network_heading">IP version</string>
```

Reply:
The Settings screen has call-to-action string. `Choose` and `Enable` etc are valid text titles. In fact, having call-to-action strings are recommended by Material UI guidelines.